### ISSUE TITLE
Explore option to paste asciidoc html input into Google Docs and have the script send back a non destructive version

### ISSUE THREAD

**Issue by emmanuelbernard** [maxandersen/gdoc2adoc#13]
Use case: write text in asciidoc and have people modify the document on Google Docs. Then send back changes to the asciidoc "source".

**Comment by maxandersen** [comment]
I believe if you take the html output of asciidoc and import into google docs is already working pretty well (except maybe TOC's). Did it fail for you ?

**Comment by emmanuelbernard** [comment]
Yes but the use case is a full loop, not a one way trip.  Adoc -> Google Docs -> Adoc without losing data.
